lib/chef/provider/mount/linux.rb

```diff
@@ -45,6 +45,13 @@ def mounted?
             when /\A#{Regexp.escape(real_mount_point)}\s+#{device_mount_regex}\s/
               mounted = true
               logger.trace("Special device #{device_logstring} mounted as #{real_mount_point}")
+            # Permalink for loop type devices mount point https://rubular.com/r/a0bS4p2RvXsGxx
+            when %r{\A#{Regexp.escape(real_mount_point)}\s+\/dev\/loop+[0-9]+\s}
+              shell_out!("losetup --list").stdout.each_line do |mount_point|
```

Comment:
we shouldn't run `losetup --list` for every single line in `findmnt`. If we have 100 mounts, we could run it 100 times and get the same output every time. We should run it just once. We could have a method to get this and memoize it in a instance var.

Reply:
Hi, How about this comparison ? Will this work ?